front/lib/api/assistant/agent_usage.ts

```diff
@@ -242,45 +243,31 @@ export async function agentMentionsUserCount(
 export async function agentMentionsCount(
   workspaceId: number
 ): Promise<mentionCount[]> {
-  // We retrieve mentions from conversations in order to optimize the query
-  // Since we need to filter out by workspace id, retrieving mentions first
-  // would lead to retrieve every single messages
-  const mentions = await Conversation.findAll({
-    attributes: [
-      [
-        Sequelize.literal('"messages->mentions"."agentConfigurationId"'),
-        "agentConfigurationId",
-      ],
-      [
-        Sequelize.fn("COUNT", Sequelize.literal('"messages->mentions"."id"')),
-        "count",
-      ],
-    ],
+  const agentConfigurations = await AgentConfiguration.findAll({
+    attributes: ["sId"],
     where: {
       workspaceId: workspaceId,
     },
-    include: [
-      {
-        model: Message,
-        required: true,
-        attributes: [],
-        where: {
-          createdAt: {
-            [Op.gt]: literal("NOW() - INTERVAL '30 days'"),
-          },
-        },
-        include: [
-          {
-            model: Mention,
-            as: "mentions",
-            required: true,
-            attributes: [],
-          },
-        ],
-      },
+  });
+  const allAgentConfigurationIds = [
+    ...agentConfigurations.map((agentConfiguration) => agentConfiguration.sId),
+    ...Object.values(GLOBAL_AGENTS_SID),
+  ];
+
+  const mentions = await Mention.findAll({
+    attributes: [
+      "agentConfigurationId",
+      [Sequelize.fn("COUNT", Sequelize.col("agentConfigurationId")), "count"],
```

Comment:
not good at sequelize enough to know if this will match the sql query that was tested in front; could you log the query when testing so we can check? (boolean to activate in front_sequelize)

Reply:
and post there the 2 queries generated?

Reply:
Sure thing! Adding the `isActive` filter first and will add the log here 👍🏼